frontend/src/scenes/insights/insightLogic.ts

```diff
@@ -138,8 +138,8 @@ export const insightLogic = kea<insightLogicType>([
             scene: Scene | null
             exception?: Record<string, any>
         }) => payload,
-        setShowTimeoutMessage: (showTimeoutMessage: boolean) => ({ showTimeoutMessage }),
-        setShowErrorMessage: (showErrorMessage: boolean) => ({ showErrorMessage }),
+        insightTimedOut: (timedOutQueryId: string | null) => ({ timedOutQueryId }),
+        insightErrored: (erroredQueryId: string | null) => ({ erroredQueryId }),
```

Comment:
Honestly this form `fooDidSomething` looks totally like a bool reducer and not an action. Maybe `markInsightTimeout`/`markInsightError`?